packages/indexer-agent/src/agent.ts

```diff
@@ -795,16 +806,32 @@ class Agent {
     )
 
     // Don't proceed if the POI is 0x0 or null
+    // with the new graph-node, deployment syncing or fatal undeterministic error
```

Comment:
This comment isn't 100% clear to me. 

Maybe something like: "A null PoI means the deployment is either not fully synced or experienced a fatal undeterministic error"